include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -211,7 +211,23 @@ __joint_inclusive_scan(_Args&&... __args)
 }
 
 #if _ONEDPL_FPGA_DEVICE
-#    if _ONEDPL_LIBSYCL_VERSION >= 50300
+#    if _ONEDPL_LIBSYCL_VERSION >= 60100
```

Comment:
I confirm it is a right value applied to Intel® oneAPI DPC++/C++ Compiler (which is a component in Intel® oneAPI Toolkits).
The latest, oneAPI 2023.1, release provides the new selectors with `ONEDPL_LIBSYCL_VERSION` equal to `60100`. 
The previous, oneAPI 2023.0, release does not provide the new selectors with `_ONEDPL_LIBSYCL_VERSION` equal to `60000`.